cfgov/ask_cfpb/models/pages.py

```diff
@@ -56,3 +62,23 @@ def __str__(self):
             return '{}: {}'.format(self.answer_base.id, self.title)
         else:
             return self.title
+
+    @property
+    def status_string(self):
```

Comment:
Can I make a request that we handle `redirect_id` specially but then call `super(AnswerPage, self).status_string` if not redirected? This will remove duplicate logic about scheduled/drafted changes.